### ISSUE TITLE
Server side rendering

### ISSUE THREAD

**Issue by Immortalin** [wasp-lang/wasp#183]
**Is your feature request related to a problem? Please describe.**
Pre render react UI server side before sending to client.

**Describe the solution you'd like**
Add Next.js as a target

**Comment by Martinsos** [comment]
Makes sense! I am somewhat reluctant about making Next our target though, since it is (I think?) not as flexible as an Express app, but I am not ruling it out as an option completely. What if we wanted to have SSR in ExpressJS app? Is there an easy way to go about that?

**Comment by samos123** [comment]
I would need server side rendering for SEO. It should be relatively easy to enable SSR out of the box like Next.js while still providing flexibility.

**Comment by Martinsos** [comment]
@juanreyero also asked if we could support SSR in the future, for SEO.
